client-v2/src/selectors/frontsSelectors.js

```diff
@@ -48,4 +70,72 @@ const frontsIdSelector = createSelector([getFronts], (fronts): string[] => {
     .sort();
 });
 
-export { frontsIdSelector, getFrontsWithPriority, getCollections };
+const getFrontsConfig = (
+  fronts: FrontConfigMap,
+  collections: CollectionConfigMap,
+  frontIds: Array<string>,
+  priority: string
+): {
+  fronts: FrontConfig[],
+  collections: CollectionConfigMap
+} => {
+  if (frontIds.length === 0) {
+    return { fronts: [], collections: {} };
+  }
+  const frontsWithPriority = frontIds.reduce(
+    (acc: Array<FrontConfig>, key: string) => {
+      if (
+        fronts[key].priority === priority ||
+        (!fronts[key].priority && priority === 'editorial')
+      ) {
+        return [...acc, fronts[key]];
+      }
+      return acc;
+    },
+    []
+  );
+  return {
+    fronts: frontsWithPriority,
+    collections
```

Comment:
Is the scope of this selector too big? It just passes through `collections` from argument to return. Could we not split this into two selectors? This is now doing the same as `getFrontsWithPriority` and `getCollections` combined?

Reply:
This is a pretty lazy port of logic that (IIRC) previously lived in a render function somewhere, and I haven't taken the time to refactor it; arguably this belongs in a separate PR, but I'm happy to do that here also if needed.